package/obs/rmt-server.spec

```diff
@@ -392,6 +392,9 @@ fi
 
 %post pubcloud
 %service_add_post rmt-server-regsharing.service rmt-server-trim-cache.service
+if [ $"(ls %{data_dir}/engines/data_export/lib/data_export/handlers/ | grep '*.rb' | wc -l > 1)" ]; then
+    mv %{data_dir}/engines/data_export/lib/data_export/handlers/example.rb %{data_dir}/engines/data_export/lib/data_export/handlers/example
+fi
```

Comment:
We could also just ship the example handler as `handler.example`, so it's not picked up by default? Moving files that got installed by rpm before seems not optimal to me, because the `example` file might not get removed on rpm removal, and a file that was installed by the package (`example.rb`) is suddenly missing.

Reply:
I like the idea

That approach has a downside, if one installs RMT with `handler.example`, RMT will start successfully 
but it will pollute the logs with "missing handler" error

Reply:
I can update the README of that engine for users that do not use/implement data export engine mentioning to chaneg the name to `handler.rb` or some name

Thoughts ?

Reply:
How about writing a warning only once, when the engine starts and there is no data export handler defined. 
After that we could just not try to do any data export.